.travis.yml

```diff
@@ -1,10 +1,26 @@
 language: node_js
 node_js:
-  - "0.10"
-  - "0.12"
   - "4"
-sudo: false
+  - "6"
+sudo: required
 before_script:
-  - npm run init
+- npm install -g web-component-tester mocha bower
+- bower install
+- npm run build
+env:
+  global:
+  - secure: X5NX0AXb+mMSzhFjvwwB/qwmH3bgiDn61XZIQjzNyM8rMvapfPOnWmYvODhwyE5jOcic7kD37uc0LjF/s8u+ZLp1UYyLNxeumIi+t6Z4fjpVGDD9MGFAHsPH4T+T3i5jFR4WMB/V8fFY4Ok/rxmJ0wbf0eOmIYKfCC1xA8lLBIQ=
+  - secure: aHvp2CDHPAHbv2QG/CBfDQBWbE2OkBH/TFJDoi2fLl0a9CDTxvxjUXPOZImnNoeLNIlN7KgOTPCQjVuvs9N5CKvJC0h3YQQ4UaAG5wQky3dXNxCUViGTn7ZHeBr9fJBzXJIwBo9ynrcAAJAWOI3ugdPewRUw92VJFbbAntQwAw0=
+addons:
+  firefox: '46.0'
+  apt:
+    sources:
+    - google-chrome
+    packages:
+    - google-chrome-stable
+  sauce_connect: true
 script:
-  - npm run testNode
+- mocha
+- xvfb-run wct
+- if [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then wct -s 'default'; fi
```

Comment:
Anyone know how to only run the Sauce tests for one of the node versions? No need to run them twice.